packages/pages/src/vite-plugin/build/closeBundle/manifest.ts

```diff
@@ -52,8 +53,12 @@ export const generateManifestFile = async (
   }
   const manifest: Manifest = {
     serverPaths: Object.fromEntries(relativeBundlePaths),
-    clientPaths: Object.fromEntries(await getClientPaths(projectStructure)),
-    renderPaths: Object.fromEntries(relativeRenderPaths),
+    clientPaths: Object.fromEntries(
```

Comment:
Can serverPaths above use the new func too?

Reply:
Decided not too as serverPaths needs to get the data from templateModules in order to get featureName rather than fileName as the key.